spec/factories/user.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+
+  factory :user do
+    email 'user@mail.ru'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.